hardware_accelerators/gaudi-ai-accelerator.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="gaudi-ai-accelerator"]
+= Intel Gaudi AI accelerators
+include::_attributes/common-attributes.adoc[]
+:context: gaudi-ai-accelerator
+
+toc::[]
+
+:FeatureName: Intel Gaudi AI accelerators
+
+You can use Intel Gaudi AI accelerators for your {product-title} generative AI and machine learning (AI/ML) applications. Intel Gaudi AI accelerators offer a cost-efficient, flexible, and scalable solution for optimized deep learning workloads.
+
+Gaudi architecture includes integrated RDMA over Converged Ethernet (RoCE v2) that allow you to use a consistent scaling technology inside the server and rack and across racks. 
```

Comment:
Do we want to include references to Gaudi RDMA when, AFAIK, it has not been officially tested or supported on OpenShift or RHOAI.

Reply:
@egallen @LaVLaS Removed the RDMA note.